### ISSUE TITLE
Accessibility blurb missing from main Application Fundamentals page

### ISSUE THREAD

**Issue by Crwth** [MicrosoftDocs/xamarin-docs#546]
The Table of Contents shows an Accessibility section under Application Fundamentals, but there is no matching mention in the landing page for Application Fundamentals.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6da1abc6-98f5-64ba-1332-d7b8f4791d22
* Version Independent ID: 522b9fe2-7192-5efd-50f2-861a7b91c446
* Content: [Xamarin.iOS Application Fundamentals - Xamarin](https://docs.microsoft.com/en-us/xamarin/ios/app-fundamentals/)
* Content Source: [docs/ios/app-fundamentals/index.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/ios/app-fundamentals/index.md)
* Product: **xamarin**
* GitHub Login: @bradumbaugh
* Microsoft Alias: **brumbaug**

**Comment by bradumbaugh** [comment]
Thanks @Crwth. I added this item and reordered the others to match the TOC. Should be published soon. I appreciate your help!
